include/mbedtls/md.h

```diff
@@ -151,6 +151,7 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160, /**< The RIPEMD-160 message digest. */
 } mbedtls_md_type_t;
 
+/* Note: this should be kept in sync with PSA_HASH_MAX_SIZE */
```

Comment:
Blocker: Please clarify the requirement (even with the commit message, I don't know the answer without digging into the code): does the MD constant always have to be equal to the PSA constant? At least as large, but strictly larger is ok? No larger, but strictly smaller is ok? What would the “trouble” be?

This should be validated either with a static assertion or a unit test. Or maybe it should be by construction.